### ISSUE TITLE
crash: [Quest:14212] Sacrifices

### ISSUE THREAD

**Issue by 3DViking** [MistCore/MistCore#14]
Hello,
When you tring to start event for that quest - http://www.wowhead.com/quest=14212 server crashing.. Someone have any ideas to fix crash??

thanks

**Comment by Dokman** [comment]
all thw worgen quest are bug xD

**Comment by 3DViking** [comment]
yup.. ;(

**Comment by Kittnz** [comment]
Yes they are bugged, the scripts & npc spawns and such need to be looked at. First i need to solve some lag issues, but stuck on that atm

**Comment by 3DViking** [comment]
Yup, that is a problem too crashing anytime with freeze detector but server didn't restart

**Comment by 3DViking** [comment]
ok, found the problem
Creature\* horse = me->FindNearestCreature(NPC_CROWLEY_HORSE, 100, true);

must be in IF(Creature\* horse = me->FindNearestCreature(NPC_CROWLEY_HORSE, 100, true))

**Comment by Kittnz** [comment]
If that is really the fix for that quest, please make a pull request! So we can add it.

**Comment by Dokman** [comment]
if you don't want to make a pull request you can tell us what file is for edit them and make a push with the fix

**Comment by Kittnz** [comment]
Just search the like dokman ;-) 
I just like people to add a pull request so they are included in a commit.

**Comment by Dokman** [comment]
yes that's true

But is possible that he don't know how to use git

**Comment by 3DViking** [comment]
i know, but im using local git (gitlab)
